### PR TITLE
Adds new integration [Rudd-O/homeassistant-meteoswiss]

### PR DESCRIPTION
**Pull request by Rudd-O**
This integration replaces the broken Swiss Meteo default HACS integration.

**Link to successful HACS action:**  https://github.com/Rudd-O/homeassistant-meteoswiss/actions/runs/4067493008/jobs/7004902764
**Link to successful hassfest action (if integration):**  https://github.com/Rudd-O/homeassistant-meteoswiss/actions/runs/4067545759/jobs/7005021903

### CODE REVIEW

**Comment by tjorim** [comment]
Last commit on that repo is only a week or two ago, did you even consider giving back to the original project by opening a PR with your fixes?

**Comment by Rudd-O** [comment]
@tjorim absolutely, but the original developer is nonresponsive.  I also had to fork his client library, because even though he merged a fundamental fix for his program, he never released it on PyPI.  I am all in favor of keeping the project going but that's not my call.

**Comment by Rudd-O** [comment]
Last commit was 3 weeks ago, and it did not fix the broken code that prevents people from using this straight from HACS.

Just look at the bug tracker.  Dozens of people still can't use the integration.

I could open a PR there, but I doubt my code would be accepted in a timely manner.

**Comment by Rudd-O** [comment]
Can the pipeline be dispatched please?  Thanks.

**Comment by Rudd-O** [comment]
Hi there @ludeeus would you mind checking this change out?  Tests are passing in the source branch.  Thank you.

**Comment by Rudd-O** [comment]
I'm baffled by the closure of the PR without further discussion, but if you want to keep a broken integration in the default HACS repository list, even when there's a fixed substitute, that's your call.

**Comment by tjorim** [comment]
This is not about the discussion, it's about tagging people that have a life outside of GitHub as well. I suggest you create a new PR and wait like everyone else does (you will notice other PR's have also been closed because of, e.g. requesting reviews).

**Comment by Rudd-O** [comment]
Bwahahaa.

> This is not about the discussion, it's about tagging people that have a life outside of GitHub as well.

We all do, buddy, we all do.  The rest of us aren't extras to your own flick's protagonism.

>  I suggest you create a new PR and wait like everyone else does (you will notice other PR's have also been closed because of, e.g. requesting reviews).

_Hahahaaahaha!_

How about **no**.

I don't know what's going on in your personal life, buddy, but I didn't open a PR here and request politely for review, just to be treated like a Department of Motor Vehicles blankface treats her "customers".  "GO BACK TO THE BACK OF THE LINE!"  _No thank you_, I'm not your servant, you are not granting me any sort of special license that requires me to do anything, and having your repo point to mine is frankly not worth to me jumping through your vindictive hoops twice.

By the way, I've done nothing _but_ be polite to either of you, too.  I was unaware that it was considered "wrong" here to tag maintainers.  Where I come from, and on the dozens of projects I've both built and helped maintain, I have always been *more than happy* to be tagged.  Furthermore, to close the PR *without comment* — not even to deign to utter "uh, we don't tag contributors here" — that's just rude.  Even if somehow tagging any of you was "impolite", exactly what are you people expecting us contributors to do in order to find out?  Learn your strange cultural customs by *sheer osmosis*?  "Oh, she tagged me in a PR, how rude, instaclosed and *no comment!*  That'll teach *her!"  LOL.

In all the time the two of you wasted recriminating me for an artificial transgression and closing this PR, you _could_ have taken a look at the PR and associated fixes, or you _could_ have passed it over to review another, eventually to come back here and explain *why* this needs re-review (if it does), or you _could_ have dropped a polite line saying "we don't take well to @mentions", or you _could_ have removed yourself from the notifications list for this PR.  You did neither of those perfectly reasonable things.

In your defense, it should be said, you don't _have_ to do any of those things — it is your prerogative to be curt, rude and to impose artificial costs on your "transgressors".

As it is my prerogative to tell yo **no**.

Bottom line: **you** can push a button to reopen the PR, _if you want_.  *I* am _not_ jumping through the silly artificial hoop of opening the exact same PR again.

Tip: if you have a problem with contributors and attention management, there are two ways to cope with it.  The good way is to figure out a way to deal with it, whether through better e-mail filters or notification settings management.  The bad way is to take your anger and frustration out on the contributors.  Once again, that's your call.
